x-pack/plugins/security_solution/public/management/pages/blocklist/view/blocklist.tsx

```diff
@@ -95,6 +95,12 @@ const BLOCKLIST_PAGE_LABELS: ArtifactListPageProps['labels'] = {
     'xpack.securitySolution.blocklist.emptyStatePrimaryButtonLabel',
     { defaultMessage: 'Add blocklist' }
   ),
+  searchPlaceholderInfo: i18n.translate(
+    'xpack.securitySolution.artifactListPage.searchPlaceholderInfo',
```

Comment:
should be `...blocklist.searchPlaceholderInfo`

Reply:
fixed: https://github.com/elastic/kibana/pull/128466/commits/501bda0195208f4b8bc8b1c240d371c09d8f68fb